fubectl.source

```diff
@@ -187,6 +187,46 @@ kwns() {
     watch kubectl get pod -n "$ns"
 }
 
+# [kbash] create a pod with a bash
+kbash() {
+    kubectl run --generator=run-pod/v1 my-shell --rm -i --tty --image ubuntu -- bash
```

Comment:
Can we try to make that not collide with other users in the cluster, and use `shell-$(date +%s)` instead of `my-shell`?

Reply:
fix